plugins/reports/counter/locale/it_IT/ar1.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/reports/counter/locale/en_US/ar1.xml
```

Comment:
I'm not sure what this file is -- maybe a work in progress that got accidentally added?